xml/System.Drawing.Imaging/DitherType.xml

```diff
@@ -13,7 +13,7 @@
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Algorithm for dithering images with a reduced color palette.</summary>
     <remarks>To be added.</remarks>
```

Comment:
```suggestion
    <remarks>For information about the ordered and spiral dither types, see the <see href="/windows/win32/api/gdipluspixelformats/ne-gdipluspixelformats-dithertype#remarks">remarks for the Win32 DitherType enumeration</see>. The fixed palette types mentioned there correspond to the <see cref="T:System.Drawing.Imaging.PaletteType" /> enumeration.</remarks>
    <related type="Article" href="/windows/win32/api/gdipluspixelformats/ne-gdipluspixelformats-dithertype">DitherType enumeration (gdipluspixelformats.h)</related>
```

Reply:
Makes sense, appreciate the suggestions!